buildifier/README.md

```diff
@@ -80,7 +80,7 @@ See also the [full list](../WARNINGS.md) or the supported warnings.
 
 ## Setup and usage via Bazel
 
-You can also invoke buildifier via the Bazel rule.
+On Linux you can also invoke buildifier via the Bazel rule.
```

Comment:
This rule also supports macOS so can you add that here I guess? Or just add `(not supported on Windows)` instead?

Reply:
@keith fixed